src/html5_parser/tokenizer/mod.rs

```diff
@@ -529,7 +529,7 @@ impl<'a> Tokenizer<'a> {
                         Element::Utf8('>') => {
                             if self.is_appropriate_end_token(&self.temporary_buffer) {
                                 let s: String = self.temporary_buffer.iter().collect::<String>();
-                                self.set_name_in_current_token(s);
+                                self.set_name_in_current_token(&s);
```

Comment:
Here we're allocating a `String` and then passing a reference.  I appreciate that this is a little weird.  I'm hoping we'll figure out a way to avoid the allocation in the future, although that might be hard in this case.